scripts/fix-volumes.sh

```diff
@@ -13,26 +13,47 @@ set -o posix    # more strict failures in subshells
 IFS=$'\n\t'
 # ---- End unofficial bash strict mode boilerplate
 
+if [[ "$#" -ne 1 ]]; then
+  echo "Usage: $0 [desired_owner_user]" 1>&2
+  exit 1
```

Comment:
This is OK but I'd also think about can we make a default like:

`desired_owner_user="${1:-1000:1000}"` so it does something reasonably likely to work even without an argument?